controllers/api.js

```diff
@@ -31,14 +31,18 @@ bitcoinclient.request('getblockchaininfo', false, function(err, info) {
   }
 });
 
-lightning.getInfo({}, function(err, info) {
+lightning.getInfo({}, function (err, info) {
   if (err) {
     console.error('lnd failure');
     console.dir(err);
     process.exit(3);
   }
   if (info) {
     console.info(info);
+    if (!info.testnet && info.block_height < MIN_BTC_BLOCK) {
```

Comment:
1) MIN_BTC_BLOCK is undefined
2) why do you need  this block? `if (!info.synced_to_chain) {` checks if LND is caught up

Reply:
I don't know why I added that, I'll remove it again